generation/localizedstringkit/detection.py

```diff
@@ -113,10 +113,20 @@ def _detect_strings(self, patterns: List[Tuple[Pattern, int]]) -> List[Localized
         # Find occurrences of `Localized` function calls
         matches_in_buffer: List[Tuple] = []
         for pattern, count in patterns:
-            matches_in_buffer += self._get_matches(pattern, count)
+            # Append source Pattern to tuple to discern if bundle is overridden
+            match = self._get_matches(pattern, count) + (pattern)
```

Comment:
We add the `Pattern` as the last element to the tuple to use it in comparison down below. That comparison is used to infer whether we pass in a keyExtension or bundle to the LocalizedString constuctor

Reply:
You should use a dedicated type here (or a tuple at the very least)

Reply:
Adding `Tuple`